### PR TITLE
Adds footer and minimal footer. [Fixes #1339]

### LINKED ISSUE #1339: Experimental Design Simplified VA Header and Footer
## What

- Simplified VA.gov header navigation that removes Sign In and the menu
- Simplified VA.gov footer navigation that removes link lists and superlinks
- Introduce a site section name or form title, along with subtitle, in to the VA.gov header navigation 

## Purpose

- Reduce the chances of users leaving a form or process (that doesn't support finishing the form later) before completing all the required steps.
- Reduce the cognitive load required to complete a form or process that doesn't support finishing it later, are time-sensitive and/or does not have a home within the VA.gov sitewide IA or include a related navigation pattern on the page.
- Reduce the cognitive load required to complete a multi-step process by quickly navigating users to the first heading/question and start interacting with the form. Reference: https://design.va.gov/patterns/help-users-to/complete-a-sub-task

### When to use this component 

- Forms or processes that don't support finishing it later, are time-sensitive and/or does not have a home within the VA.gov sitewide IA or include a related navigation pattern on the page. 

## Behavior

- Forms or processes using a subtitle can be included immediately after the form title in normal case. 
- Hide the VA seal on desktop screens to anchor all pages with the form title as a replacement for a traditional h1 that's persistent across all form pages.   

## Examples

[Patient Check-In prototype with simplified header/footer and progress bar - segmented
](https://www.sketch.com/s/a720996f-5ebb-4092-8e5d-8e3dcff50dc7/prototype/a/E50121F2-F405-46C7-9FE6-8505525F7010)

[Desktop version](https://www.sketch.com/s/a720996f-5ebb-4092-8e5d-8e3dcff50dc7/a/L0LVoDQ)

## Accessibility

- The form title should take on the banner landmark for assistive technologies. A banner landmark role overwrites the implicit ARIA role for the container element upon which it is applied. It should be reserved for globally repeating sitewide content that is generally located at the top of every page. 

## Guidance

N/a

### Research that has already been conducted

- 0.4% of users within a 30-day period interacted with any of the VA.gov footer navigation superlinks while using VA's Pre-Check-In product. In addition, 6.6% of those same users interacted with the VA.gov header menu and 5% (6,555 of 131,136 sessions) with the VA.gov Sign In, respectively, which in most scenarios resulted in an unsuccessful Pre-Check-In. The Patient Check In (PCI) product does not support full authentication, and therefore the presence of VA.gov Sign In increases the chances of Veterans leaving the Pre-Check-In process (as well as Check-In) before completing all the required steps. Reference: [Header and footer events for patient check in.xlsx](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/files/9983720/Header.and.footer.events.for.patient.check.in.xlsx)
- Average page load time for Check-In pages (11.19 seconds) is almost triple than the VA.gov average (4.08 seconds) according to [Google Analytics](https://analytics.google.com/analytics/web/#/report/content-site-speed/a50123418w177519031p176188361/_u.dateOption=last30days&explorer-table.advFilter=%5B%5B0,%22analytics.pagePath%22,%22RE%22,%22appointment-check-in%22,0%5D%5D&explorer-table.plotKeys=%5B%5D/). In a time-sensitive interaction like checking in for an appointment, page load time is of utmost importance to success. Therefore, the Modernized Check-In Experience Team hypothesizes that the number of check-ins will increase if the VA.gov header and footer files are not loaded within the PCI product, which would decrease the average page load time.  
- A Veteran in a [recent research study for Patient Check-In](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/research/veteran-facing/Spanish/Spanish%20Check-in%20%26%20Toggle%20Design%20Research%20Report.pdf) said, "The top where it says sign in and menu could be confusing to those that are just trying to sign in to an appointment." 

### Research we plan to conduct

TBD

## Code (optional)

N/a

### PR DESCRIPTION
**Pull request by humancompanion-usds**

### CODE REVIEW

File: src/_components/footer/minimal.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+* **Reduce the chance of a user leaving a form, process, or task before completing the required steps.** The minimal footer retains VA branding so that users still know they are on VA.gov. However, it removes elements of the standard footer, such additional navigation, in order to reduce the chances of users leaving a form or process before completing all the required steps and accomplishing their goal. While most forms support finishing the form later, we still prefer to keep users focused on the task at hand.
+* **A tasks that is time-sensitive.** The minimal footer is also appropriate when a user must complete a task in a timely fashion and thus navigating away would put task completion at risk.
+* **An experience outside of the VA.gov site-wide information architecture.** Some experiences do not exist within the information architecture of VA.gov or have already implemented a distinct navigation pattern and thus can also use the minimal footer.
+
+### How this component works
+
+* **Use with the [Header - Minimal]({{ site.baseurl }}/components/header/header-minimal).** The minimal footer should be used in conjunction with the minimal header.
```
Comment (danbrady) [inline src/_components/footer/minimal.md:35]: Do we want to update the Minimal Header page to have a similar sentence regarding it should always be used with the Minimal Footer?

File: src/_includes/component-resource-links.html [hunk 2]
```diff
@@ -43,6 +43,13 @@
       </a>
     </dd>
     {% endif %}
+    {% if page.figma-link %}
+    <dd role="definition" class="site-component-resources-links__definition">
+      <a class="site-component-resources-links__link" href="{{page.figma-link}}" target="_blank" title="View the visual design for this component (internal only)">
```
[the remark this answers is not in the document]
Comment (humancompanion-usds) [reply src/_includes/component-resource-links.html:48]: So this warning goes away, yes.

File: src/_components/footer/minimal.md (hunk 1 above)
Comment (humancompanion-usds) [reply src/_components/footer/minimal.md:35]: Yes, good call.

**Review by danbrady — APPROVED** [review]
LGTM

**Pull request merged**